build.gradle.kts

```diff
@@ -36,9 +35,18 @@ plugins {
 
 buildscript {
     standardSpineSdkRepositories()
+
     dependencies {
-        @Suppress("RemoveRedundantQualifierName")
-        classpath(io.spine.internal.dependency.Spine.McJava.pluginLib)
+        classpath(variantOf(spine.mcJava) { classifier("all") })
+    }
+
+    configurations.all {
+        resolutionStrategy.force(
+            spine.protoData.plugin,
+            spine.protoData.compiler,
+            spine.protoData.codegenJava,
+            spine.validation.codegenJava,
```

Comment:
Again, `Validation` is also a top-level dependency object now. Please migrate.

Reply:
Same as above.